internal/store/postgres/lineage_repository.go

```diff
@@ -177,9 +179,9 @@ func (repo *LineageRepository) removeGraph(ctx context.Context, execer sqlx.Exec
 	return nil
 }
 
-func (repo *LineageRepository) compareGraph(current, new asset.LineageGraph) (toInserts, toRemoves asset.LineageGraph) {
+func (*LineageRepository) compareGraph(current, new asset.LineageGraph) (toInserts, toRemoves asset.LineageGraph) {
```

Comment:
here - the named returns are still present. Expected ?

Reply:
Named returns are meant to be used as documentation when there are multiple values with identical types being returned. So this is intentional.